packages/icons/src/Icon.tsx

```diff
@@ -38,7 +38,10 @@ const Icon = ({
   const isAriaHidden = ariaHidden ?? (!ariaLabelledBy && !ariaLabel);
   const titleId = title && `${prefix}-${name}-title`;
   const descriptionId = description && `${prefix}-${name}-description`;
-  const { path: spritePath } = useContext(IconContext);
+
+  const { path: contextPath } = useContext(IconContext);
+  const iconId = `lp-icon-${name}`;
```

Comment:
Could this ID be part of the context value, and then either provide a path or an ID to that context?

Reply:
True we can have it default to `lp-icon` but open it up to custom ones too 👍

Reply:
Oh sorry, I actually had a fundamental misunderstanding of what this ID was doing. I thought this was the ID for the sprite SVG, not the individual icon within that sprite.

I guess when it's injected into the DOM, we don't need to know the sprite's ID, since each icon has its own ID that we can reference globally.